contracts/courtroom_test.go

```diff
@@ -194,36 +201,43 @@ func openClaimForReflectorGame(t *testing.T, clientContent string, serviceConten
 	if err != nil {
 		t.Fatalf("balanceOfClientBefore : expected no error, got %v", err)
 	}
-	if clientContent != serviceContent {
-		if amountStakedBefore.Int64()-amountStakedAfter.Int64() != 5 {
-			t.Fatalf("After a valid claim proccess : amountStaked at the contract should reduce by 5  ( got %d)", (amountStakedBefore.Int64() - amountStakedAfter.Int64()))
+	if (clientContent != serviceContent) && (numberOfBlocksToWait <= PromiseTillNextBlocks) {
```

Comment:
@zelig why this should be in contract ? The contract just check that the service done (or not) his work within the blocks period it promised .
The test here just wait X blocks to test a none valid claim.(which is not according what the service promised)
I will comment that on the test